pkg/role/sdkserviceapi.go

```diff
@@ -74,8 +74,8 @@ var (
 					"credentials",
 					"objectstore",
 					"schedulepolicy",
-					"volume",
 					"mountattach",
+					"migrate",
```

Comment:
Was this removed on purpose?

Reply:
Yeah, github hides it, but that slice of strings actually has `"volume"` twice in there

Reply:
we noticed it when looking through this code in the meeting. @lpabon said to remove it.

Reply:
Yeah ok cool,  yeah I remember now.   Was just checking.